interface/forms/gad7/.directory

```diff
@@ -0,0 +1,4 @@
+[Dolphin]
+Timestamp=2020,7,22,11,50,9
+Version=4
+ViewMode=1
```

Comment:
hi @ruthkonyn , thanks again for working your way thru to this stage, going to provide some comments here to start the review.

Like this file, if it's nonfunctional maybe better to put it in the header of the gad7 `new.php` since not used to seeing hidden files in the other forms directories.

Reply:
apologies, the .directory file is not needed, I should have added it to .ignore, it can be deleted.

Reply:
should be the way to go, a `checkout -b <my new branch>` and you can make your changes and start a new PR then you can [recreate your master](https://github.com/openemr/openemr/pull/657#issuecomment-295630669)

Reply:
thanks stephen - i've just created a new branch 'donate_to_OpenEMR', pushed to github and sumitted brand new PR which I guess you'll see now checks are completed.